### PR TITLE
Flesh out descriptions

### PR DESCRIPTION
**Pull request by rgbkrk**
Attempts to start down the UX path from #156

### CODE REVIEW

**Comment by rgbkrk** [comment]
Done making my edits. Would love feedback on the technical accuracy and helpfulness of the commands. It's important to look at what these appear like on the command line rather than how they read in source (they're called "usage" here, but they appear as a description on the cli):

```
$ rack block-storage
NAME:
   rack block-storage - Block level storage, exposed as volumes to mount to host servers. Work with volumes and their associated snapshots.

USAGE:
   rack block-storage command [command options] [arguments...]

COMMANDS:
   snapshot	Copies of block storage volumes at a specific moment in time. Use for backups, restores, and long term storage.
   volume	Block level volumes to expand storage on your servers.
   help, h	Shows a list of commands or help for one command

OPTIONS:
   --help, -h			show help
   --generate-bash-completion
```

File: commands/blockstoragecommands/commands.go [hunk 1]
```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "snapshot",
-			Usage:       "Used for BlockStorage Snapshot operations",
+			Usage:       "Copies of block storage volumes at a specific moment in time. Use for backups, restores, and long term storage.",
```
Comment (kenperkins) [inline commands/blockstoragecommands/commands.go]: perhaps `used for backup, restoration, and other long term storage`

File: commands/blockstoragecommands/commands.go [hunk 2]
```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
...
 			Subcommands: snapshotcommands.Get(),
 		},
 		{
 			Name:        "volume",
-			Usage:       "Used for BlockStorage Volume operations",
+			Usage:       "Block level volumes to expand storage on your servers.",
```
Comment (kenperkins) [inline commands/blockstoragecommands/commands.go]: `Block level volumes used to add storage capacity on your servers`? I worry that "expand" could be a bit confusing.

File: commands/filescommands/commands.go [hunk 3]
```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "container",
-			Usage:       "Used for Cloud Files container operations",
+			Usage:       "Storage compartments for your objects/files.",
```
Comment (kenperkins) [inline commands/filescommands/commands.go]: compartment feels wrong here. Why not use the name itself?

`Containers for storing objects in the cloud`

File: commands/blockstoragecommands/commands.go (hunk 2 above)
Comment (rgbkrk) [inline commands/blockstoragecommands/commands.go]: You're right, expand could be convoluted to mean the volume itself which can't happen (IIRC).

File: commands/filescommands/commands.go [hunk 4]
```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
...
 			Subcommands: containercommands.Get(),
 		},
 		{
 			Name:        "object",
-			Usage:       "Used for Cloud Files object operations",
+			Usage:       "Data storage for objects/files/media.",
```
Comment (kenperkins) [inline commands/filescommands/commands.go]: I also think this is a bit confusing. `Data storage` feels a bit off. Struggling to come up with a better description however.

File: commands/filescommands/commands.go (hunk 3 above)
Comment (rgbkrk) [inline commands/filescommands/commands.go]: I was trying my hardest to not be cyclic, and I stole it straight from Rackspace's docs about containers.

File: commands/networkscommands/commands.go [hunk 5]
```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "network",
-			Usage:       "Used for Cloud Networks network operations",
+			Usage:       "Set up new networks to use by servers.",
```
Comment (kenperkins) [inline commands/networkscommands/commands.go]: `Set up new networks` is a bit of a tongue twister.

Thinking outloud. `Software defined networks used by Cloud Servers`

File: commands/networkscommands/commands.go [hunk 6]
```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
...
 		{
 			Name:        "subnet",
-			Usage:       "Used for Cloud Networks subnet operations",
+			Usage:       "Allocate IP address blocks, gateways, DNS servers, and host routes to networks.",
 			Subcommands: subnetcommands.Get(),
 		},
 		{
 			Name:        "port",
-			Usage:       "Used for Cloud Networks port operations",
+			Usage:       "Virtual switch ports on logical network switches.",
```
Comment (kenperkins) [inline commands/networkscommands/commands.go]: I don't know what this means from this description.

File: commands/filescommands/commands.go (hunk 3 above)
Comment (rgbkrk) [inline commands/filescommands/commands.go]: How about just "Containers for storing objects."

File: commands/networkscommands/commands.go [hunk 7]
```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
...
 		{
 			Name:        "security-group",
-			Usage:       "Used for Cloud Networks security group operations",
+			Usage:       "Collections of rules for network traffic.",
 			Subcommands: securitygroupcommands.Get(),
 		},
 		{
 			Name:        "security-group-rule",
-			Usage:       "Used for Cloud Networks security group rule operations",
+			Usage:       "Rules for network traffic.",
```
Comment (kenperkins) [inline commands/networkscommands/commands.go]: I think this should not be pluralized, as it's an individual rule. Perhaps: `A rule used to define network ingress and egress`

File: commands/networkscommands/commands.go (hunk 6 above)
Comment (rgbkrk) [inline commands/networkscommands/commands.go]: ...yeah I didn't either

http://docs.rackspace.com/servers/api/v2/cn-devguide/content/conceptPort.html

I've not used any of these features. Some of them are in early availability only.

File: commands/filescommands/commands.go (hunk 3 above)
Comment (kenperkins) [inline commands/filescommands/commands.go]: I like that actually.

File: commands/networkscommands/commands.go (hunk 5 above)
Comment (rgbkrk) [inline commands/networkscommands/commands.go]: Updating this to your description. Wish there was some way to say this isn't the only stop - they have to set up the subnet in a separate command here.

File: commands/networkscommands/commands.go (hunk 7 above)
Comment (rgbkrk) [inline commands/networkscommands/commands.go]:
```
$ rack networks security-group-rule
NAME:
   rack networks security-group-rule - Define network ingress and egress rules.
```

**Comment by rgbkrk** [comment]
I'm realizing these all need to start out lowercase to fit in with the format:

```
NAME:
  rack command subcommand - description of service
```

File: commands/networkscommands/commands.go (hunk 7 above)
Comment (kenperkins) [inline commands/networkscommands/commands.go]: :+1:

File: main.go [hunk 8]
```diff
@@ -35,32 +35,32 @@ OPTIONS:
...
 		{
 			Name:        "files",
-			Usage:       "Used for the Files service",
+			Usage:       "Object storage for files and media.",
 			Subcommands: filescommands.Get(),
 		},
 		{
 			Name:        "networks",
-			Usage:       "Used for the Networks service",
+			Usage:       "Software defined networking.",
```
Comment (jrperritt) [inline main.go]: s/Software defined/Software-defined

Comment (rgbkrk) [inline main.go]: Wow, without the dash it can mean "Software - it defined networking"

File: commands/networkscommands/commands.go [hunk 9]
```diff
@@ -14,27 +14,27 @@ func Get() []cli.Command {
 	return []cli.Command{
 		{
 			Name:        "network",
-			Usage:       "Used for Cloud Networks network operations",
+			Usage:       "Software defined networks used by servers.",
```
Comment (jrperritt) [inline commands/networkscommands/commands.go]: s/Software defined/Software-defined

File: commands/blockstoragecommands/commands.go [hunk 10]
```diff
@@ -11,12 +11,12 @@ func Get() []cli.Command {
...
 		{
 			Name:        "snapshot",
-			Usage:       "Used for BlockStorage Snapshot operations",
+			Usage:       "Copies of block storage volumes at a specific moment in time. Used for backup, restoration, and other long term storage.",
 			Subcommands: snapshotcommands.Get(),
 		},
 		{
 			Name:        "volume",
-			Usage:       "Used for BlockStorage Volume operations",
+			Usage:       "Block level volumes to add storage capacity to your servers.",
```
Comment (jrperritt) [inline commands/blockstoragecommands/commands.go]: s/Block level/Block-level

**Comment by jrperritt** [comment]
I don't have anything to add here. These are all significantly better than what was there before.

**Comment by rgbkrk** [comment]
Addressed the dashes.

**Pull request merged**